### ISSUE TITLE
link to demo of embedding neuroglancer in a jupyter notebook

### ISSUE THREAD

**Issue by d-v-b** [google/neuroglancer#676]
see https://github.com/holoviz-topics/neuro/blob/main/workflows/neuroglancer_notebook/neuroglancer-nb-workflow.ipynb. I think this workflow could be very attractive to a lot of people.

**Comment by d-v-b** [comment]
it might make sense to link to this demo from somewhere in the python docs

**Comment by jbms** [comment]
Yes, this seems like it would be of interest to a lot of people --- I'd be happy to take a PR that adds this to the "Related projects" section of the README, or potentially to the Python API docs somewhere though those docs are still kind of rough...

**Comment by kabilar** [comment]
Thanks, I have filed #994.

It looks like the new link is available here: https://examples.holoviz.org/gallery/volumetric_imaging/volumetric_imaging.html

And the source code is available here: https://github.com/holoviz-topics/neuro/blob/2357155dcb2b795754488cb2b313144f009d1b8c/workflows/volumetric_imaging/volumetric_imaging.ipynb
